docs/content/components/hero.mdx

```diff
@@ -89,6 +89,12 @@ best_practices: |
     - Aspect Ratio - 1:1
     - Minimum Upload Size - 640px x 640px (@2x retina)
 
+  ### External Links
+  For CTA Strip buttons that have links outside of .gov and other supported sites, an external link icon will be added at the end of the button's title to indicate that the link will go to an external site. This icon is added automatically based on where the link the is going to, but can also be overridden to display or be hidden regardless of the link.
+
+  - To override an INTERNAL link to be displayed as EXTERNAL, add the class `ncids-external-link` to the anchor (`<a>`) element.
```

Comment:
`usa-link` uses `usa-link--external`, the footer pr used `external-link` this one is using `ncids-external-link`